packages/xo-web/src/xo-app/hub/resource.js

```diff
@@ -207,6 +219,17 @@ export default decorate([
       <CardBlock className='text-center'>
         <div>
           <span className='text-muted'>{_('os')}</span> <strong>{os}</strong>
+          {description !== undefined && (
+            <span className='pull-right'>
+              <a
+                className='text-secondary'
+                style={{ cursor: 'pointer' }}
+                onClick={effects.showDescription}
+              >
+                <Icon icon='info' /> {_('vmNameDescription')}
+              </a>
```

Comment:
I think It would be simpler and best for accessibility to use a button for this.

Links should be reserved to *link* to other pages via URLs.